### ISSUE TITLE
Name the installation package differently than the executable file of the installed application

### ISSUE THREAD

**Issue by Korb** [shaked6540/YoutubePlaylistDownloader#383]
Historically, it's common for an installation package to have the word `install` or `setup` in its filename. Otherwise, it might appear that the application consists of a single executable file (yes, this happens - take the Sysinternals Suite, not to mention Windows mini-apps). And the requirement for the Internet loader to operate with administrative rights is a wake-up call for the moderately concerned Windows user (and almost all Linux users).

Also, there may be such situations:

![2024-10-16_16-35-16](https://github.com/user-attachments/assets/c205059a-bfd2-4896-90bf-7549bb7be9e9)

**Comment by shaked6540** [comment]
You are free to change the installer name to whatever you want after you download it

The requirement for administrator privileges is not up to me, you can run it as a regular user if you change the permissions of the folder you try to download the files into. By default windows only allows administrator to write files into C drive, except some special folders (like appdata, temp, etc)

**Comment by cristianst85** [comment]
Were you trying to install it while the application was already running?

Also, I already explained https://github.com/shaked6540/YoutubePlaylistDownloader/issues/238#issuecomment-2071146543 that is should work using the default settings and without running it with Administrator privileges.

**Comment by Korb** [comment]
> Were you trying to install it while the application was already running?

No.

> Also, I already explained here that is should work using the default settings and without running it with Administrator privileges.

You didn't get me, and maybe I didn't make it clear enough. I'll describe step by step how the process might look from the user's point of view:

1. Download `YoutubePlaylistDownloader.exe`
2. Create a folder for the application.
3. Since the binary name doesn't include `setup` or `install` (or version number), move `YoutubePlaylistDownloader.exe` to the created folder.
4. Run `YoutubePlaylistDownloader.exe` so that the application creates a user settings file, including the path to `yt-dlp.exe` (after automatically searching or manually specifying).
5. The user suddenly receives a request for Administrator privileges. He doesn’t know that this is not a portable application, as the name might suggest, but an installer.
6. If the user wisely did not refuse to grant privileges, he specifies the path for installing the application. The folder that he created in the second step.
7. Gets the error shown in the screenshot in the first post. Because, suddenly, the installer name is exactly the same as the file name of the unpacked application.

However, never mind. Since no one has had this problem in six years, fixing the causes isn't worth the effort.

**Comment by cristianst85** [comment]
When you download VLC (or any other app) you also do steps 2 or 3?

**Comment by Korb** [comment]
> When you download VLC (or any other app) you also do steps 2 and 3?

Yes.

I started compiling a list of applications installed on my PC for you, when updating which this problem does not arise due to obvious names of installers or archives (if for updating it is enough to unpack files and overwrite old ones), and I came to the conclusion that this is almost all applications. In all cases, I clearly understand when I downloaded a package that requires installation, and when I downloaded a portable application - without having to launch it just to find out! You decided to complicate the task of users to find out. I do not understand your reasoning. To be precise, there was none.

I am not sure that listing dozens of application names or developers here will change anything in terms of your attitude to the generally accepted practice, which you decided not to adhere to. This is your right, but you risk wasting time before coming to the same conclusions as many before you.
